[2017-03-01 19:33:50] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:33:50] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:33:50] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:33:50] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:33:50] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:33:50] app.INFO: An AuthenticationException was thrown; redirecting to authentication entry point. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AuthenticationCredentialsNotFoundException(code: 0): A Token was not found in the TokenStorage. at C:\\Users\\Zach\\spatter\\vendor\\symfony\\security\\Http\\Firewall\\AccessListener.php:53)"} []
[2017-03-01 19:33:50] app.DEBUG: Calling Authentication entry point. [] []
[2017-03-01 19:33:50] app.DEBUG: < 302 /auth/login [] []
[2017-03-01 19:33:50] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 19:33:50] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:33:50] app.DEBUG: > GET /auth/login [] []
[2017-03-01 19:33:50] app.DEBUG: < 200 [] []
[2017-03-01 19:39:34] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 19:39:34] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:39:34] app.DEBUG: > GET /auth/login [] []
[2017-03-01 19:39:34] app.DEBUG: < 200 [] []
[2017-03-01 19:39:40] app.INFO: Matched route "{route}". {"route":"users_login","route_parameters":{"_controller":null,"_route":"users_login"},"request_uri":"http://localhost:8888/users/login","method":"POST"} []
[2017-03-01 19:39:40] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:39:40] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:39:40] app.DEBUG: Passing guard token information to the GuardAuthenticationProvider {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:39:41] app.INFO: Guard authentication failed. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\UsernameNotFoundException(code: 0): Username \"zach\" does not exist. at C:\\Users\\Zach\\spatter\\app\\Providers\\UserProvider.php:25)","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:39:41] app.DEBUG: The "{authenticator}" authenticator set the response. Any later authenticator will not be called {"authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:39:41] app.DEBUG: < 302 /auth/login [] []
[2017-03-01 19:39:41] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 19:39:41] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:39:41] app.DEBUG: > GET /auth/login [] []
[2017-03-01 19:39:41] app.DEBUG: < 200 [] []
[2017-03-01 19:40:08] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 19:40:08] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:40:08] app.DEBUG: > GET /auth/login [] []
[2017-03-01 19:40:09] app.DEBUG: < 200 [] []
[2017-03-01 19:40:46] app.INFO: Matched route "{route}". {"route":"users_login","route_parameters":{"_controller":null,"_route":"users_login"},"request_uri":"http://localhost:8888/users/login","method":"POST"} []
[2017-03-01 19:40:46] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:40:46] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:40:46] app.DEBUG: Passing guard token information to the GuardAuthenticationProvider {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:40:46] app.INFO: Guard authentication failed. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\UsernameNotFoundException(code: 0): Username \"test\" does not exist. at C:\\Users\\Zach\\spatter\\app\\Providers\\UserProvider.php:25)","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:40:46] app.DEBUG: The "{authenticator}" authenticator set the response. Any later authenticator will not be called {"authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:40:46] app.DEBUG: < 302 /auth/login [] []
[2017-03-01 19:40:46] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 19:40:46] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:40:46] app.DEBUG: > GET /auth/login [] []
[2017-03-01 19:40:46] app.DEBUG: < 200 [] []
[2017-03-01 19:40:59] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 19:40:59] app.DEBUG: Read existing security token from the session. {"key":"_security_register"} []
[2017-03-01 19:40:59] app.CRITICAL: RuntimeException: There is no user provider for user "App\Models\User". (uncaught exception) at C:\Users\Zach\spatter\vendor\symfony\security\Http\Firewall\ContextListener.php line 179 {"exception":"[object] (RuntimeException(code: 0): There is no user provider for user \"App\\Models\\User\". at C:\\Users\\Zach\\spatter\\vendor\\symfony\\security\\Http\\Firewall\\ContextListener.php:179)"} []
[2017-03-01 19:40:59] app.DEBUG: < 500 [] []
[2017-03-01 19:41:26] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 19:41:26] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:41:26] app.DEBUG: > GET /auth/register [] []
[2017-03-01 19:41:26] app.DEBUG: < 200 [] []
[2017-03-01 19:41:38] app.INFO: Matched route "{route}". {"route":"POST_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"POST_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"POST"} []
[2017-03-01 19:41:38] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:41:38] app.DEBUG: > POST /auth/register [] []
[2017-03-01 19:41:39] app.DEBUG: < 200 [] []
[2017-03-01 19:41:51] app.INFO: Matched route "{route}". {"route":"POST_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"POST_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"POST"} []
[2017-03-01 19:41:51] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:41:51] app.DEBUG: > POST /auth/register [] []
[2017-03-01 19:41:52] app.DEBUG: < 302 / [] []
[2017-03-01 19:41:52] app.DEBUG: Stored the security token in the session. {"key":"_security_register"} []
[2017-03-01 19:41:52] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:41:52] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:41:52] app.DEBUG: User was reloaded from a user provider. {"username":"zach","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:41:52] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:41:52] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:41:52] app.DEBUG: > GET / [] []
[2017-03-01 19:41:52] app.DEBUG: < 200 [] []
[2017-03-01 19:41:52] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:42:00] app.INFO: Matched route "{route}". {"route":"POST_post_","route_parameters":{"_controller":"App\\Controller\\PostController::store","_route":"POST_post_"},"request_uri":"http://localhost:8888/post/","method":"POST"} []
[2017-03-01 19:42:00] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:42:00] app.DEBUG: User was reloaded from a user provider. {"username":"zach","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:42:00] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:42:00] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:42:00] app.DEBUG: > POST /post/ [] []
[2017-03-01 19:42:00] app.DEBUG: < 302 / [] []
[2017-03-01 19:42:00] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:42:00] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:42:00] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:42:00] app.DEBUG: User was reloaded from a user provider. {"username":"zach","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:42:00] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:42:00] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:42:00] app.DEBUG: > GET / [] []
[2017-03-01 19:42:00] app.CRITICAL: Twig_Error_Runtime: Neither the property "created_at" nor one of the methods "created_at()", "getcreated_at()"/"iscreated_at()" or "__call()" exist and have public access in class "App\Models\Post". (uncaught exception) at C:\Users\Zach\spatter\public\views\pages\index.html.twig line 19 {"exception":"[object] (Twig_Error_Runtime(code: 0): Neither the property \"created_at\" nor one of the methods \"created_at()\", \"getcreated_at()\"/\"iscreated_at()\" or \"__call()\" exist and have public access in class \"App\\Models\\Post\". at C:\\Users\\Zach\\spatter\\public\\views\\pages\\index.html.twig:19)"} []
[2017-03-01 19:42:00] app.DEBUG: < 500 [] []
[2017-03-01 19:42:00] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:43:09] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:43:09] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:43:09] app.CRITICAL: Doctrine\DBAL\Exception\InvalidFieldNameException: An exception occurred while executing 'SELECT u0_.id AS id_0, u0_.username AS username_1, u0_.email AS email_2, u0_.password AS password_3, u0_.createdAt AS createdAt_4, u0_.updatedAt AS updatedAt_5, u0_.is_active AS is_active_6 FROM users u0_ WHERE u0_.username = ?' with params ["zach"]:  SQLSTATE[42S22]: Column not found: 1054 Unknown column 'u0_.createdAt' in 'field list' (uncaught exception) at C:\Users\Zach\spatter\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 71 {"exception":"[object] (Doctrine\\DBAL\\Exception\\InvalidFieldNameException(code: 0): An exception occurred while executing 'SELECT u0_.id AS id_0, u0_.username AS username_1, u0_.email AS email_2, u0_.password AS password_3, u0_.createdAt AS createdAt_4, u0_.updatedAt AS updatedAt_5, u0_.is_active AS is_active_6 FROM users u0_ WHERE u0_.username = ?' with params [\"zach\"]:\n\nSQLSTATE[42S22]: Column not found: 1054 Unknown column 'u0_.createdAt' in 'field list' at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:71, Doctrine\\DBAL\\Driver\\PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'u0_.createdAt' in 'field list' at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:93, PDOException(code: 42S22): SQLSTATE[42S22]: Column not found: 1054 Unknown column 'u0_.createdAt' in 'field list' at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOStatement.php:91)"} []
[2017-03-01 19:43:09] app.DEBUG: < 500 [] []
[2017-03-01 19:43:35] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:43:35] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:43:35] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:35] app.INFO: An AuthenticationException was thrown; redirecting to authentication entry point. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AuthenticationCredentialsNotFoundException(code: 0): A Token was not found in the TokenStorage. at C:\\Users\\Zach\\spatter\\vendor\\symfony\\security\\Http\\Firewall\\AccessListener.php:53)"} []
[2017-03-01 19:43:35] app.DEBUG: Calling Authentication entry point. [] []
[2017-03-01 19:43:35] app.DEBUG: < 302 /auth/login [] []
[2017-03-01 19:43:35] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 19:43:35] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:43:35] app.DEBUG: > GET /auth/login [] []
[2017-03-01 19:43:35] app.DEBUG: < 200 [] []
[2017-03-01 19:43:43] app.INFO: Matched route "{route}". {"route":"users_login","route_parameters":{"_controller":null,"_route":"users_login"},"request_uri":"http://localhost:8888/users/login","method":"POST"} []
[2017-03-01 19:43:43] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:43:43] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:43] app.DEBUG: Passing guard token information to the GuardAuthenticationProvider {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:44] app.INFO: Guard authentication failed. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\BadCredentialsException(code: 0): Authentication failed because App\\Security\\LoginAuthenticator::checkCredentials() did not return true. at C:\\Users\\Zach\\spatter\\vendor\\symfony\\security\\Guard\\Provider\\GuardAuthenticationProvider.php:126)","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:44] app.DEBUG: The "{authenticator}" authenticator set the response. Any later authenticator will not be called {"authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:44] app.DEBUG: < 302 /auth/login [] []
[2017-03-01 19:43:44] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 19:43:44] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:43:44] app.DEBUG: > GET /auth/login [] []
[2017-03-01 19:43:44] app.DEBUG: < 200 [] []
[2017-03-01 19:43:48] app.INFO: Matched route "{route}". {"route":"users_login","route_parameters":{"_controller":null,"_route":"users_login"},"request_uri":"http://localhost:8888/users/login","method":"POST"} []
[2017-03-01 19:43:48] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:43:48] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:48] app.DEBUG: Passing guard token information to the GuardAuthenticationProvider {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:49] app.INFO: Guard authentication successful! {"token":"[object] (Symfony\\Component\\Security\\Guard\\Token\\PostAuthenticationGuardToken: PostAuthenticationGuardToken(user=\"zach\", authenticated=true, roles=\"ROLE_USER\"))","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:49] app.DEBUG: Guard authenticator set success response. {"response":"[object] (Symfony\\Component\\HttpFoundation\\RedirectResponse: HTTP/1.0 302 Found\r\nCache-Control: no-cache, private\r\nLocation:      /\r\n\r\n<!DOCTYPE html>\n<html>\n    <head>\n        <meta charset=\"UTF-8\" />\n        <meta http-equiv=\"refresh\" content=\"1;url=/\" />\n\n        <title>Redirecting to /</title>\n    </head>\n    <body>\n        Redirecting to <a href=\"/\">/</a>.\n    </body>\n</html>)","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:49] app.DEBUG: Remember me skipped: your authenticator does not support it. {"authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:49] app.DEBUG: The "{authenticator}" authenticator set the response. Any later authenticator will not be called {"authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:49] app.DEBUG: < 302 / [] []
[2017-03-01 19:43:49] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:43:49] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:43:49] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:43:49] app.DEBUG: User was reloaded from a user provider. {"username":"zach","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:43:49] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:43:49] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:43:49] app.DEBUG: > GET / [] []
[2017-03-01 19:43:49] app.CRITICAL: Twig_Error_Runtime: Neither the property "created_at" nor one of the methods "created_at()", "getcreated_at()"/"iscreated_at()" or "__call()" exist and have public access in class "App\Models\Post". (uncaught exception) at C:\Users\Zach\spatter\public\views\pages\index.html.twig line 19 {"exception":"[object] (Twig_Error_Runtime(code: 0): Neither the property \"created_at\" nor one of the methods \"created_at()\", \"getcreated_at()\"/\"iscreated_at()\" or \"__call()\" exist and have public access in class \"App\\Models\\Post\". at C:\\Users\\Zach\\spatter\\public\\views\\pages\\index.html.twig:19)"} []
[2017-03-01 19:43:49] app.DEBUG: < 500 [] []
[2017-03-01 19:43:49] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:46:50] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:46:50] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:46:50] app.DEBUG: User was reloaded from a user provider. {"username":"zach","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:46:50] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:46:50] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:46:50] app.DEBUG: > GET / [] []
[2017-03-01 19:46:50] app.CRITICAL: Twig_Error_Runtime: Neither the property "created_at" nor one of the methods "created_at()", "getcreated_at()"/"iscreated_at()" or "__call()" exist and have public access in class "App\Models\Post". (uncaught exception) at C:\Users\Zach\spatter\public\views\pages\index.html.twig line 19 {"exception":"[object] (Twig_Error_Runtime(code: 0): Neither the property \"created_at\" nor one of the methods \"created_at()\", \"getcreated_at()\"/\"iscreated_at()\" or \"__call()\" exist and have public access in class \"App\\Models\\Post\". at C:\\Users\\Zach\\spatter\\public\\views\\pages\\index.html.twig:19)"} []
[2017-03-01 19:46:50] app.DEBUG: < 500 [] []
[2017-03-01 19:46:50] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:47:11] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:47:11] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:47:11] app.WARNING: Username could not be found in the selected user provider. {"username":null,"provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:47:11] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:47:11] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:47:11] app.INFO: An AuthenticationException was thrown; redirecting to authentication entry point. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\AuthenticationCredentialsNotFoundException(code: 0): A Token was not found in the TokenStorage. at C:\\Users\\Zach\\spatter\\vendor\\symfony\\security\\Http\\Firewall\\AccessListener.php:53)"} []
[2017-03-01 19:47:11] app.DEBUG: Calling Authentication entry point. [] []
[2017-03-01 19:47:11] app.DEBUG: < 302 /auth/login [] []
[2017-03-01 19:47:11] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 19:47:11] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:47:11] app.DEBUG: > GET /auth/login [] []
[2017-03-01 19:47:11] app.DEBUG: < 200 [] []
[2017-03-01 19:47:14] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 19:47:14] app.DEBUG: Read existing security token from the session. {"key":"_security_register"} []
[2017-03-01 19:47:14] app.CRITICAL: RuntimeException: There is no user provider for user "App\Models\User". (uncaught exception) at C:\Users\Zach\spatter\vendor\symfony\security\Http\Firewall\ContextListener.php line 179 {"exception":"[object] (RuntimeException(code: 0): There is no user provider for user \"App\\Models\\User\". at C:\\Users\\Zach\\spatter\\vendor\\symfony\\security\\Http\\Firewall\\ContextListener.php:179)"} []
[2017-03-01 19:47:14] app.DEBUG: < 500 [] []
[2017-03-01 19:48:39] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 19:48:39] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:48:39] app.DEBUG: > GET /auth/register [] []
[2017-03-01 19:48:40] app.DEBUG: < 200 [] []
[2017-03-01 19:49:19] app.INFO: Matched route "{route}". {"route":"POST_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"POST_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"POST"} []
[2017-03-01 19:49:19] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 19:49:19] app.DEBUG: > POST /auth/register [] []
[2017-03-01 19:49:20] app.DEBUG: < 302 / [] []
[2017-03-01 19:49:20] app.DEBUG: Stored the security token in the session. {"key":"_security_register"} []
[2017-03-01 19:49:20] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:49:20] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:49:20] app.DEBUG: User was reloaded from a user provider. {"username":"zachary","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:49:20] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:49:20] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:49:20] app.DEBUG: > GET / [] []
[2017-03-01 19:49:20] app.DEBUG: < 200 [] []
[2017-03-01 19:49:20] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:49:28] app.INFO: Matched route "{route}". {"route":"POST_post_","route_parameters":{"_controller":"App\\Controller\\PostController::store","_route":"POST_post_"},"request_uri":"http://localhost:8888/post/","method":"POST"} []
[2017-03-01 19:49:28] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:49:28] app.DEBUG: User was reloaded from a user provider. {"username":"zachary","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:49:28] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:49:28] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:49:28] app.DEBUG: > POST /post/ [] []
[2017-03-01 19:49:28] app.DEBUG: < 302 / [] []
[2017-03-01 19:49:28] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:49:28] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:49:28] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:49:28] app.DEBUG: User was reloaded from a user provider. {"username":"zachary","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:49:28] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:49:28] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:49:28] app.DEBUG: > GET / [] []
[2017-03-01 19:49:28] app.CRITICAL: Twig_Error_Runtime: Neither the property "created_at" nor one of the methods "created_at()", "getcreated_at()"/"iscreated_at()" or "__call()" exist and have public access in class "App\Models\Post". (uncaught exception) at C:\Users\Zach\spatter\public\views\pages\index.html.twig line 19 {"exception":"[object] (Twig_Error_Runtime(code: 0): Neither the property \"created_at\" nor one of the methods \"created_at()\", \"getcreated_at()\"/\"iscreated_at()\" or \"__call()\" exist and have public access in class \"App\\Models\\Post\". at C:\\Users\\Zach\\spatter\\public\\views\\pages\\index.html.twig:19)"} []
[2017-03-01 19:49:28] app.DEBUG: < 500 [] []
[2017-03-01 19:49:28] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:49:43] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:49:43] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:49:43] app.DEBUG: User was reloaded from a user provider. {"username":"zachary","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:49:43] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:49:43] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:49:43] app.DEBUG: > GET / [] []
[2017-03-01 19:49:44] app.DEBUG: < 200 [] []
[2017-03-01 19:49:44] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 19:51:07] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 19:51:07] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 19:51:07] app.DEBUG: User was reloaded from a user provider. {"username":"zachary","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 19:51:07] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 19:51:07] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 19:51:07] app.DEBUG: > GET / [] []
[2017-03-01 19:51:07] app.DEBUG: < 200 [] []
[2017-03-01 19:51:07] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 20:00:22] app.INFO: Matched route "{route}". {"route":"users_logout","route_parameters":{"_controller":null,"_route":"users_logout"},"request_uri":"http://localhost:8888/users/logout","method":"GET"} []
[2017-03-01 20:00:22] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 20:00:22] app.CRITICAL: Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) (uncaught exception) at C:\Users\Zach\spatter\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)"} []
[2017-03-01 20:00:22] app.DEBUG: < 500 [] []
[2017-03-01 20:01:02] app.INFO: Matched route "{route}". {"route":"users_logout","route_parameters":{"_controller":null,"_route":"users_logout"},"request_uri":"http://localhost:8888/users/logout","method":"GET"} []
[2017-03-01 20:01:02] app.DEBUG: < 302 http://localhost:8888/auth/login [] []
[2017-03-01 20:01:02] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 20:01:02] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:01:02] app.DEBUG: > GET /auth/login [] []
[2017-03-01 20:01:02] app.DEBUG: < 200 [] []
[2017-03-01 20:01:04] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 20:01:04] app.DEBUG: Read existing security token from the session. {"key":"_security_register"} []
[2017-03-01 20:01:04] app.CRITICAL: RuntimeException: There is no user provider for user "App\Models\User". (uncaught exception) at C:\Users\Zach\spatter\vendor\symfony\security\Http\Firewall\ContextListener.php line 179 {"exception":"[object] (RuntimeException(code: 0): There is no user provider for user \"App\\Models\\User\". at C:\\Users\\Zach\\spatter\\vendor\\symfony\\security\\Http\\Firewall\\ContextListener.php:179)"} []
[2017-03-01 20:01:04] app.DEBUG: < 500 [] []
[2017-03-01 20:01:10] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 20:01:10] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:01:10] app.DEBUG: > GET /auth/register [] []
[2017-03-01 20:01:10] app.DEBUG: < 200 [] []
[2017-03-01 20:01:12] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 20:01:12] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:01:12] app.DEBUG: > GET /auth/register [] []
[2017-03-01 20:01:12] app.DEBUG: < 200 [] []
[2017-03-01 20:01:13] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 20:01:13] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:01:13] app.DEBUG: > GET /auth/register [] []
[2017-03-01 20:01:13] app.DEBUG: < 200 [] []
[2017-03-01 20:01:14] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 20:01:14] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:01:14] app.DEBUG: > GET /auth/login [] []
[2017-03-01 20:01:14] app.DEBUG: < 200 [] []
[2017-03-01 20:01:15] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 20:01:15] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:01:15] app.DEBUG: > GET /auth/register [] []
[2017-03-01 20:01:15] app.DEBUG: < 200 [] []
[2017-03-01 20:01:16] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 20:01:16] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:01:16] app.DEBUG: > GET /auth/login [] []
[2017-03-01 20:01:16] app.DEBUG: < 200 [] []
[2017-03-01 20:01:22] app.INFO: Matched route "{route}". {"route":"users_login","route_parameters":{"_controller":null,"_route":"users_login"},"request_uri":"http://localhost:8888/users/login","method":"POST"} []
[2017-03-01 20:01:22] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 20:01:22] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:01:22] app.DEBUG: Passing guard token information to the GuardAuthenticationProvider {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:01:22] app.CRITICAL: Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) (uncaught exception) at C:\Users\Zach\spatter\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)"} []
[2017-03-01 20:01:22] app.DEBUG: < 500 [] []
[2017-03-01 20:01:39] app.INFO: Matched route "{route}". {"route":"users_login","route_parameters":{"_controller":null,"_route":"users_login"},"request_uri":"http://localhost:8888/users/login","method":"POST"} []
[2017-03-01 20:01:39] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 20:01:39] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:01:39] app.DEBUG: Passing guard token information to the GuardAuthenticationProvider {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:01:39] app.CRITICAL: Doctrine\DBAL\Exception\ConnectionException: An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) (uncaught exception) at C:\Users\Zach\spatter\vendor\doctrine\dbal\lib\Doctrine\DBAL\Driver\AbstractMySQLDriver.php line 103 {"exception":"[object] (Doctrine\\DBAL\\Exception\\ConnectionException(code: 0): An exception occured in driver: SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\AbstractMySQLDriver.php:103, Doctrine\\DBAL\\Driver\\PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:47, PDOException(code: 1045): SQLSTATE[HY000] [1045] Access denied for user 'root'@'localhost' (using password: YES) at C:\\Users\\Zach\\spatter\\vendor\\doctrine\\dbal\\lib\\Doctrine\\DBAL\\Driver\\PDOConnection.php:43)"} []
[2017-03-01 20:01:39] app.DEBUG: < 500 [] []
[2017-03-01 20:01:59] app.INFO: Matched route "{route}". {"route":"users_login","route_parameters":{"_controller":null,"_route":"users_login"},"request_uri":"http://localhost:8888/users/login","method":"POST"} []
[2017-03-01 20:01:59] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 20:01:59] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:01:59] app.DEBUG: Passing guard token information to the GuardAuthenticationProvider {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:01:59] app.INFO: Guard authentication failed. {"exception":"[object] (Symfony\\Component\\Security\\Core\\Exception\\UsernameNotFoundException(code: 0): Username \"zach\" does not exist. at C:\\Users\\Zach\\spatter\\app\\Providers\\UserProvider.php:25)","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:01:59] app.DEBUG: The "{authenticator}" authenticator set the response. Any later authenticator will not be called {"authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:01:59] app.DEBUG: < 302 /auth/login [] []
[2017-03-01 20:01:59] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 20:01:59] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:01:59] app.DEBUG: > GET /auth/login [] []
[2017-03-01 20:02:00] app.DEBUG: < 200 [] []
[2017-03-01 20:02:10] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 20:02:10] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:02:10] app.DEBUG: > GET /auth/register [] []
[2017-03-01 20:02:10] app.DEBUG: < 200 [] []
[2017-03-01 20:02:22] app.INFO: Matched route "{route}". {"route":"POST_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"POST_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"POST"} []
[2017-03-01 20:02:22] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:02:22] app.DEBUG: > POST /auth/register [] []
[2017-03-01 20:02:23] app.DEBUG: < 302 / [] []
[2017-03-01 20:02:23] app.DEBUG: Stored the security token in the session. {"key":"_security_register"} []
[2017-03-01 20:02:23] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 20:02:23] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 20:02:23] app.DEBUG: User was reloaded from a user provider. {"username":"lisavo","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 20:02:23] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 20:02:23] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:02:23] app.DEBUG: > GET / [] []
[2017-03-01 20:02:23] app.DEBUG: < 200 [] []
[2017-03-01 20:02:23] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 20:02:34] app.INFO: Matched route "{route}". {"route":"POST_post_","route_parameters":{"_controller":"App\\Controller\\PostController::store","_route":"POST_post_"},"request_uri":"http://localhost:8888/post/","method":"POST"} []
[2017-03-01 20:02:34] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 20:02:34] app.DEBUG: User was reloaded from a user provider. {"username":"lisavo","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 20:02:34] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 20:02:34] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:02:34] app.DEBUG: > POST /post/ [] []
[2017-03-01 20:02:34] app.DEBUG: < 302 / [] []
[2017-03-01 20:02:34] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 20:02:35] app.INFO: Matched route "{route}". {"route":"GET_","route_parameters":{"_controller":"App\\Controller\\PageController::home","_route":"GET_"},"request_uri":"http://localhost:8888/","method":"GET"} []
[2017-03-01 20:02:35] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 20:02:35] app.DEBUG: User was reloaded from a user provider. {"username":"lisavo","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 20:02:35] app.DEBUG: Checking for guard authentication credentials. {"firewall_key":"main","authenticators":1} []
[2017-03-01 20:02:35] app.DEBUG: Calling getCredentials() on guard configurator. {"firewall_key":"main","authenticator":"App\\Security\\LoginAuthenticator"} []
[2017-03-01 20:02:35] app.DEBUG: > GET / [] []
[2017-03-01 20:02:35] app.DEBUG: < 200 [] []
[2017-03-01 20:02:35] app.DEBUG: Stored the security token in the session. {"key":"_security_main"} []
[2017-03-01 20:02:37] app.INFO: Matched route "{route}". {"route":"users_logout","route_parameters":{"_controller":null,"_route":"users_logout"},"request_uri":"http://localhost:8888/users/logout","method":"GET"} []
[2017-03-01 20:02:37] app.DEBUG: Read existing security token from the session. {"key":"_security_main"} []
[2017-03-01 20:02:37] app.DEBUG: User was reloaded from a user provider. {"username":"lisavo","provider":"App\\Providers\\UserProvider"} []
[2017-03-01 20:02:37] app.DEBUG: Clearing remember-me cookie. {"name":"REMEMBERME"} []
[2017-03-01 20:02:37] app.DEBUG: < 302 http://localhost:8888/auth/login [] []
[2017-03-01 20:02:37] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 20:02:37] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:02:37] app.DEBUG: > GET /auth/login [] []
[2017-03-01 20:02:37] app.DEBUG: < 200 [] []
[2017-03-01 20:02:41] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 20:02:41] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:02:41] app.DEBUG: > GET /auth/login [] []
[2017-03-01 20:02:41] app.DEBUG: < 200 [] []
[2017-03-01 20:02:41] app.INFO: Matched route "{route}". {"route":"GET_auth_register","route_parameters":{"_controller":"App\\Controller\\AuthController::registerUser","_route":"GET_auth_register"},"request_uri":"http://localhost:8888/auth/register","method":"GET"} []
[2017-03-01 20:02:41] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:02:41] app.DEBUG: > GET /auth/register [] []
[2017-03-01 20:02:42] app.DEBUG: < 200 [] []
[2017-03-01 20:02:42] app.INFO: Matched route "{route}". {"route":"GET_auth_login","route_parameters":{"_controller":"App\\Controller\\AuthController::showLoginForm","_route":"GET_auth_login"},"request_uri":"http://localhost:8888/auth/login","method":"GET"} []
[2017-03-01 20:02:42] app.INFO: Populated the TokenStorage with an anonymous Token. [] []
[2017-03-01 20:02:42] app.DEBUG: > GET /auth/login [] []
[2017-03-01 20:02:43] app.DEBUG: < 200 [] []
